setup_warehouse.py

```
@@ -0,0 +1,59 @@
+import os
+import django
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'ddpui.settings')
+django.setup()
+
+from ddpui.models.org import Org, OrgWarehouse
+from ddpui.models.org_user import OrgUser
```

Ruff (0.12.2)

`ddpui.models.org_user.OrgUser` imported but unused

Remove unused import: `ddpui.models.org_user.OrgUser`

(F401)